src/routes/index.jsx

```diff
@@ -13,6 +17,9 @@ const Routes = () => (
       <Route exact path="/" component={HomeView} />
       <Route path="/profile" component={UserProfile} />
       <Route path="/reset-password/:resetToken" component={ResetPassword} />
+      <Route path="/login" component={Login} />
+      <ProtectedRoute path="/article/create" component={CreateArticleView} />
+      <ProtectedRoute path="/articles/drafts" component={DraftsView} />
```

Comment:
@RachelleMaina, protect these routes from being accessed by a user that is not logged in, connect this component with redux, pick the state that determines a user's login status, make a condition that renders that route based on that state.